src/components/intern-app.html

```diff
@@ -78,46 +78,60 @@ <h1 title="Intern App">Intern App</h1>
    <template is="dom-if" if="[[!addUserSelected]]">
      <button class="add-button" type="button" on-click="createNewUserProfile">[[changeButtonText(addUserSelected)]]<jha-icon-person-add></jha-icon-person-add></button>
    </template>
-   <user-list user-profiles="[[userProfiles]]" add-user-selected="[[addUserSelected]]" local-storage-class="[[localStorageClass]]"></user-list>
+   <user-list user-profiles="[[userProfiles]]" add-user-selected="[[addUserSelected]]" firebase-class="[[firebaseClass]]"></user-list>
    <template is="dom-if" if="[[cardsPopulated]]">
       <button class="delete-all-button" type="button" on-click="deleteAll">Delete All Users<jha-icon-delete></jha-icon-delete></button>
    </template>
  </template>
  <script type="module">
-   'use strict';
-   import localStorageApi from "../api/local-storage.js";
-   
-   class InternApp extends Polymer.Element {
-     static get is() { return 'intern-app'; }
+  'use strict';
+  import firebase from "firebase/app";
+  import "firebase/database";
+  import firebaseApi from "../api/firebase-api.js";
+  
+   // Initialize Firebase
+  const firebaseConfig = {
+    apiKey: "AIzaSyAF2m66MvU-JyOkS4Q7u4YH10Vf30zT2tU",
```

Comment:
it's bad practice to leave commit your keys like this. it's a bit of a PITA to store it elsewhere and add it back in every time you need to connect, but it's a helluva lot safer.